### PR TITLE
Tutorials improvements

### PR DESCRIPTION
**Pull request by DominiqueMakowski**
- [ ] Improve advanced
- [ ] Add MLE example
- [x] Show an example of a mixed model (but to not sample because it's too slow for now)

### CODE REVIEW

**Comment by DominiqueMakowski** [comment]
If you want feel free to merge that and I'll open a new PR once for the next updates

File: docs/src/turing_simple.md [hunk 1]
```diff
@@ -4,21 +4,23 @@ It is possible to use [Turing.jl](https://turinglang.org/stable/) to perform Bay
...
 
-```@setup turing_simple
+```@example turing_simple
 using Turing
 using SequentialSamplingModels
 using Random
 using LinearAlgebra
 using StatsPlots
-
-Random.seed!(45461)
```
Comment (itsdfish) [inline docs/src/turing_simple.md:21]: I think the rng seed should be visible to readers so they can reproduce the results in the documentation.

File: docs/src/rdm.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
 # Racing Diffusion Model
 
-The Diffusion Race Model (DRM; Tillman, Van Zandt, & Logan, 2020) is a sequential sampling model in which evidence for options races independently. The DRM is similar to the Linear Ballistic Accumulator, except it assumes noise occurs during the within-trial evidence accumulation process, but the drift rate is constant across trials. 
+The DRacing Diffusion Model (RDM; Tillman, [Van Zandt, & Logan, 2020](https://link.springer.com/article/10.3758/s13423-020-01719-6)) is a sequential sampling model in which evidence for options races independently. The DRM is similar to the Linear Ballistic Accumulator, except it assumes noise occurs during the within-trial evidence accumulation process, but the drift rate is constant across trials.
```
Comment (itsdfish) [inline docs/src/rdm.md:3]: Thanks for catching my error. 

It looks like your correction has an extraneous `D`

File: docs/src/rdm.md [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
 
 # Example
-In this example, we will demonstrate how to use the DRM in a generic two alternative forced choice task. 
+In this example, we will demonstrate how to use the DRM in a generic two alternative forced choice task.
```
Comment (itsdfish) [inline docs/src/rdm.md:6]: It looks like the wrong acronym is used throughout (my apologies). Can you do a find and replace on DRM and drm?

Comment (DominiqueMakowski) [reply docs/src/rdm.md]: should be fixed

Comment (itsdfish) [reply docs/src/rdm.md]: Thanks!

**Pull request merged**